ui/components/multichain/pages/page/page.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { Box } from '../../../component-library';
+import {
+  BlockSize,
+  Display,
+  FlexDirection,
+} from '../../../../helpers/constants/design-system';
+
+export const Page = ({
```

Comment:
I was thinking about the location of`Page` here. I feel like we can use it in various flows and should be put outside `multichain` folder. wdyt?

Reply:
Eventually in concert with the DesignSystem team, but it's only going to be used in Multichain components for now, so it makes sense to have here.